src/main/java/com/paypal/test/DBUpdate.java

```diff
@@ -0,0 +1,38 @@
+package com.paypal.test;
+
+import java.io.BufferedReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.util.stream.Stream;
+
+public class DBUpdate {
+    void updateDB() throws Exception {
+        String url="jdbc:mysql://prod-host:3306/testdb?useSSL=false";
+        String user="testuser";
```

Comment:
Needed a Global authentication table to validate the user credentials for growing user base.